docs/commands/watch.md

```diff
@@ -106,6 +106,8 @@ Your function MUST have the `apigw_http` feature enabled in the `lambda_http` de
 
 If your project includes several functions under the same package, you can access them using the function's name as the prefix in the request path `http://localhost:9000/lambda-url/FUNCTION_NAME`. You can also add any additional path after the function name, or any query parameters.
 
+You can specify which function(s) to include by using the `--bin` flag. If you only specify one function, then you can access it directly using the root path `http://localhost:9000`.
```

Comment:
this should probably be in the section below, which explains the same concept with the `--package` flag. You can just say that both flags work in the same way.

Reply:
This scenario/feat is specific to multi-function packages. For a user with a single package project, I doubt they will read a section titled "specific packages".

Reply:
Ok, I'll rework this section of the docs, because it's intended to explain exactly this use case. I just ran a quick smoke test and everything seems to work as expected. Thanks for the contribution!

Reply:
Thanks for the amazing package <3